### ISSUE TITLE
OAS ja kaavaselostus

### ISSUE THREAD

**Issue by SannaJokela** [sykefi/kaavatietomalli#100]
Tietojärjestelmänäkökulmasta OAS- ja kaavaselostustaulut ovat lähes identtisiä. Onko tarve yhdistää ne ja koodilistan kautta valita onko liitetty taulun rivi OAS vai kaavaselostus? Tietojärjestelmätasolla näitä voisi käsitellä samalla tavoin. Tämä ei ole kriittinen huomio.

**Comment by ilkkarinne** [comment]
Joo, nykyisellään ovat enemmänkin placeholder-tyyppisiä luokkia, jotka lisäämällä on varauduttu tulevaisuuden rakenteisiin OAS- ja selostusluokkiin.
